src/components/templates/PartyMap/components/Map/PartyMapRoomOverlay.tsx

```diff
@@ -14,104 +17,205 @@ import RoomAttendance from "../RoomAttendance";
 interface PartyMapRoomOverlayProps {
   // Passed down from Map component
   room: PartyMapRoomData;
+  enterRoom: (room: PartyMapRoomData) => void;
 
   // Passed down from Camp component (via Map component)
   venue: PartyMapVenue;
   attendances: Attendances;
   selectedRoom?: PartyMapRoomData;
-  setSelectedRoom: (room?: PartyMapRoomData) => void;
-  setIsRoomModalOpen: (value: boolean) => void;
+  selectRoom: (room: CampRoomData) => void;
+  unselectRoom: () => void;
 }
 
 export const PartyMapRoomOverlay: React.FC<PartyMapRoomOverlayProps> = ({
   room,
   venue,
   attendances,
   selectedRoom,
-  setSelectedRoom,
-  setIsRoomModalOpen,
+  selectRoom,
+  unselectRoom,
 }) => {
   const isSelectedRoom = room === selectedRoom;
+
+  // TODO: can we cleanup/de-dupe these attendance/attendees things?
+  const attendees =
+    (attendances[`${venue.name}/${room.title}`] ?? 0) +
+    (room.attendanceBoost ?? 0);
   const hasAttendance = attendances[`${venue.name}/${room.title}`];
+  const hasAttendees = attendees > 0;
```

Comment:
- [ ] [TODO] can we cleanup/de-dupe these attendance/attendees things?

Reply:
Chris: The attendance boost was related to counting issues. We can probably get rid of this bit here.

`attendances` is keyed by `venue` or `venue/room` and has a count.

Reply:
We can probably centralise the counting logic into redux, so that we only need to do it in one place.

Reply:
`attendances`/related removed in https://github.com/sparkletown/sparkle/pull/840/commits/0e39c82b5ce1444a476728279b03f9005b0e14e6